Redmine 5.1-stable branch error in v3

During supporting Redmine 6.1 in redmine_custom_fields_groups plugin with this action v3, I encountered the following error.
https://github.com/gtt-project/redmine_custom_fields_groups/actions/runs/19800337649/job/56726687487
```
Redmine Version: 5.1.10.stable
Redmine Major Version Number: 501
Run if [ $REDMINE_MAJOR_VERSION_NUMNER -lt $MINIUMUM_SUPPORTED_REDMINE_VERSION ]; then
  if [ $REDMINE_MAJOR_VERSION_NUMNER -lt $MINIUMUM_SUPPORTED_REDMINE_VERSION ]; then
    echo "Redmine version $REDMINE_VERSION is not supported."
    exit 1
  fi
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    PLUGIN_NAME: redmine_custom_fields_groups
    RAILS_ENV: test
    REDMINE_VERSION: 5.1.10.stable
    REDMINE_VERSION_MAJOR: 5
    REDMINE_VERSION_MINOR: 1
    REDMINE_VERSION_TINY: 10
    REDMINE_VERSION_BRANCH: stable
    REDMINE_MAJOR_VERSION_NUMNER: 501
    MINIUMUM_SUPPORTED_REDMINE_VERSION: 510
Redmine version 5.1.10.stable is not supported.
Error: Process completed with exit code 1.
```

From the following comment, I guess that `MINIUMUM_SUPPORTED_REDMINE_VERSION` may need to be changed from `510` to `501`, but checking the value is helpful.
https://github.com/hidakatsuya/action-setup-redmine/blob/main/scripts/set-version-envs.sh#L27-L28
```sh
# v5.1.3.stable -> 501
echo "REDMINE_MAJOR_VERSION_NUMNER=$redmine_major_version_number" >> $GITHUB_ENV
```

Also, this is slight thing, but `REDMINE_MAJOR_VERSION_NUMNER` seems to be typo (`NUMNER` => `NUMBER`), so just for your information.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -35,6 +35,13 @@ jobs:
             expected-about-db-adapter: "Mysql2"
             expected-about-redmine-version: '6\.0\.[0-9]\+\.stable'
 
+          - redmine-repository: "redmine/redmine"
+            redmine-version: "5.1-stable"
+            redmine-database: "sqlite3"
+            ruby-version: "3.2"
+            expected-about-db-adapter: "SQLite"
+            expected-about-redmine-version: '5\.1\.[0-9]\+\.stable'
+
           - redmine-repository: "redmica/redmica"
             redmine-version: "stable-4.0"
             redmine-database: "postgres:14"

--- a/action.yml
+++ b/action.yml
@@ -53,13 +53,13 @@ runs:
 
     - name: Check if the Redmine version is supported
       run: |
-        if [ $REDMINE_MAJOR_VERSION_NUMNER -lt $MINIUMUM_SUPPORTED_REDMINE_VERSION ]; then
+        if [ $REDMINE_MAJOR_VERSION_NUMBER -lt $MINIUMUM_SUPPORTED_REDMINE_VERSION ]; then
           echo "Redmine version $REDMINE_VERSION is not supported."
           exit 1
         fi
       shell: bash
       env:
-        MINIUMUM_SUPPORTED_REDMINE_VERSION: "510" # v5.1.0
+        MINIUMUM_SUPPORTED_REDMINE_VERSION: "501" # v5.1.0
 
     - name: Set up base environment
       run: $GITHUB_ACTION_PATH/scripts/setup-base.sh

--- a/scripts/set-version-envs.sh
+++ b/scripts/set-version-envs.sh
@@ -25,7 +25,7 @@ echo "REDMINE_VERSION_TINY=$version_tiny" >> $GITHUB_ENV
 echo "REDMINE_VERSION_BRANCH=$version_branch" >> $GITHUB_ENV
 
 # v5.1.3.stable -> 501
-echo "REDMINE_MAJOR_VERSION_NUMNER=$redmine_major_version_number" >> $GITHUB_ENV
+echo "REDMINE_MAJOR_VERSION_NUMBER=$redmine_major_version_number" >> $GITHUB_ENV
 
 # Output version information
 echo "Redmine Version: $redmine_version"

